server/schedulers/utils.go

```diff
@@ -190,3 +107,13 @@ func adjustBalanceLimit(cluster schedule.Cluster, kind core.ResourceKind) uint64
 	limit, _ := stats.StandardDeviation(stats.Float64Data(counts))
 	return maxUint64(1, uint64(limit))
 }
+
+const (
+	taintCacheGCInterval = time.Second * 5
+	taintCacheTTL        = time.Minute * 5
```

Comment:
Why choose 5 minutes, I see the region balance TTL less than leader balance. if the cluster reached a balanced state and then leader loss in one store, we may need to wait 5 minutes to start balance.

Reply:
I use a long TTL to make sure all stores have a chance to try balance. The problem you mentioned do exist. I fix it by clear the cache when no store can be selected to try balance. PTAL.